cfme/containers/container.py

```diff
@@ -59,21 +61,45 @@ def get_random_instances(cls, provider, count=1, appliance=None):
                 for obj in itertools.islice(containers_list, count)]
 
 
+class ContainerAllView(BaseLoggedInPage):
+
+    containers = Table(locator="//div[@id='list_grid']//table")
+
+    @View.nested
+    class filters(Accordion):  # noqa
+
+        ACCORDION_NAME = "Filters"
+
+        @View.nested
+        class navigation(VerticalNavigation):
+
+            DIV_LINKS_MATCHING = './/div/ul/li/a[contains(text(), {txt})]'
```

Comment:
@rsnyman Check this out - one way of 'fixing' the bug we were talking about today.

@gshefer rsynman is working on a widgetsatic VerticalNavigation change to support this on multiple pages instead of having to replicate your overriding here.  Plan is to define a second locator constant for contains (including href like the original one) and defining a parameter for partial matches (`VerticalNavigation(partial=True)`)